zeebe/qa/update-tests/src/test/java/io/camunda/zeebe/test/NoSnapshotTest.java

```diff
@@ -46,7 +46,6 @@ void update(
     state.withNetwork(network).withOldBroker().start(true);
     final long processInstanceKey = testCase.setUp(state.client());
     final long key = testCase.runBefore(state);
-    assertThat(state).hasNoSnapshotAvailable(1);
```

Comment:
🤔 Should we assert it's at position 0? 🤷

Reply:
I did that initially - https://github.com/camunda/camunda/pull/37202/commits/751d68c2b90dc2884479cd9527ba4297fdb17f45 

But I think we don't need to be that strict. If you want, I can bring those changes back.